ivy/functional/frontends/jax/random.py

```diff
@@ -283,3 +283,19 @@ def pareto(key, b, shape=None, dtype="float64"):
     e = -ivy.log(1 - uniform)
 
     return ivy.exp(e / b)
+
+
+@handle_jax_dtype
+@to_ivy_arrays_and_back
+@with_unsupported_dtypes(
+    {
+        "0.4.11 and below": (
+            "float16",
+            "bfloat16",
+        )
+    },
+    "jax",
+)
+def ball(key, d, p=2, shape=(), dtype="float64"):
+    seed = _get_seed(key)
+    return ivy.ball(d, p, shape=shape, dtype=dtype, seed=seed)
```

Comment:
Ivy doesn't have the ball function implemented in the functional API, you'll need to implement this using composition